eng/testing/RunnerTemplate.sh

```diff
@@ -197,4 +197,10 @@ if [[ "$(uname -s)" == "Linux" && $test_exitcode -ne 0 ]]; then
 fi
 popd >/dev/null
 # ======================== END Core File Inspection ==========================
-exit $test_exitcode
+# The helix work item should not exit with non-zero if tests ran and produced results
+# The special console runner for runtime returns 1 when tests fail
+if [ "$test_exitcode" != "1" ]; then
+  exit 0
+else
+  exit $test_exitcode
```

Comment:
Why not this, since exit $testcode can only happen if $testcode is "1" anyway

```suggestion
  exit 1
```

Reply:
The if is supposed to be ==, i'll fix it.